Graph/runTests.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+CurDir=$(dirname $(readlink -f $0))
+TestDir=$CurDir/TestsForGraph
+argumentForCheckingFails=0
+existingOfMake=0
+
+cd $TestDir
+make > /dev/null
+cd $CurDir
+
+if [[ -f "$CurDir/minDist" ]]
+then
+    existingOfMake=1
+fi
+
+make > /dev/null
```

Comment:
Testing script must not build anything, ALL you need for testing must be ready before running this script. In your Readme must be written what to do to run the tests.
"For running tests do the following:
1. cd to/my/test/dir
2. make

Afther that, run myTesterScript.sh which is located in my/tester/script/dir"